CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+-Add: kafkaCustom.headers support including the ability to override the default Fiware-Service and Fiware-ServicePath headers (#4726)
```

Comment:
```suggestion
-Fix: kafkaCustom.headers support including the ability to override the default Fiware-Service and Fiware-ServicePath headers (#4726)
```

It would be "Fix" given that is was a described feature in documentation that was not working (this is different from a new feature, i.e. in the case `headers` wasn't included in kafkaCustom noficitations documentation at the present moment, in that case "Add" would be used).